InAppBillingImplementation.cs

```diff
@@ -197,6 +197,7 @@ public override async Task<InAppBillingPurchase> PurchaseAsync(string productId,
                     purchase = await PurchaseAsync(productId, "inapp", payload, verifyPurchase);
                     break;
                 case ItemType.Subscription:
+                case ItemType.Voucher:
```

Comment:
Delete this line

Reply:
if we delete this line it will not call PurchaseAsync